tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -240,7 +240,15 @@ internal SecurityCoordinator(Security security, Span span, HttpTransport? transp
             // key could be null, but it's not a valid key in a dictionary
             // Using [] instead of Add to avoid potential duplicate key
             // but it does mean there's a (tiny) chance of overwriting the key
-            formData[key ?? string.Empty] = _httpTransport.Context.Request.Form[key];
+            try
+            {
+                formData[key ?? string.Empty] = _httpTransport.Context.Request.Form[key];
+            }
+            catch (HttpRequestValidationException)
```

Comment:
Do/should we catch this anywhere else? e.g. querystring?

Reply:
Yes, some time ago, I launched a PR to capture the same exception in querystrings. https://github.com/DataDog/dd-trace-dotnet/pull/5221

This is another case that has been spotted recently.